runbooks/schedule_backfill.ipynb

```diff
@@ -0,0 +1,444 @@
+    "        continue\n",
+    "    ts = pendulum.parse(d.split(\"/\")[-1])\n",
+    "    if first_date <= ts.date() and ts.date() < first_date_v2:\n",
+    "        folders.append(d)\n",
+    "len(folders), folders[0], folders[-1]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "80f7d728-320a-40f7-97b9-99798d171037",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import base64\n",
+    "from typing import ClassVar, List, Optional\n",
+    "\n",
+    "import pendulum\n",
+    "from calitp.storage import (\n",
+    "    GTFSDownloadConfig,\n",
+    "    GTFSFeedType,\n",
+    "    GTFSScheduleFeedExtract,\n",
+    "    PartitionedGCSArtifact,\n",
+    "    ProcessingOutcome,\n",
+    ")\n",
+    "from google.cloud import storage\n",
+    "from pydantic import HttpUrl, parse_obj_as, validator\n",
+    "\n",
+    "\n",
+    "class GTFSDownloadOutcome(ProcessingOutcome):\n",
+    "    config: GTFSDownloadConfig\n",
+    "    extract: Optional[GTFSScheduleFeedExtract]\n",
+    "    backfilled: bool = False\n",
+    "\n",
+    "\n",
+    "class DownloadFeedsResult(PartitionedGCSArtifact):\n",
+    "    bucket: ClassVar[str] = os.environ[\"CALITP_BUCKET__GTFS_SCHEDULE_RAW\"]\n",
+    "    table: ClassVar[str] = \"download_schedule_feed_results\"\n",
+    "    partition_names: ClassVar[List[str]] = [\"dt\", \"ts\"]\n",
+    "    ts: pendulum.DateTime\n",
+    "    end: pendulum.DateTime\n",
+    "    outcomes: List[GTFSDownloadOutcome] = []\n",
+    "    backfilled: bool = False\n",
+    "\n",
+    "    #@validator(\"backfilled\", allow_reuse=True)\n",
+    "    #def everything_backfilled(cls, v, values):\n",
+    "    #    outcomes_backfilled = set(outcome.backfilled for outcome in values[\"outcomes\"])\n",
+    "    #    assert {v} == outcomes_backfilled\n",
+    "    #    return v\n",
+    "    \n",
+    "    @property\n",
+    "    def dt(self) -> pendulum.Date:\n",
+    "        return self.ts.date()\n",
+    "\n",
+    "    def save(self, fs):\n",
+    "        self.save_content(\n",
+    "            fs=fs,\n",
+    "            content=\"\\n\".join(o.json() for o in self.outcomes).encode(),\n",
+    "            exclude={\"outcomes\"},\n",
+    "        )\n",
+    "        return self"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bb3c62ad-8332-46cd-9fa7-c8d3dbe8dc58",
+   "metadata": {
+    "jupyter": {
+     "source_hidden": true
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import csv\n",
+    "import io\n",
+    "import json\n",
+    "import re\n",
+    "from collections import Counter\n",
+    "from concurrent.futures import Future, ThreadPoolExecutor, as_completed\n",
+    "from pprint import pprint\n",
+    "from zipfile import ZipFile\n",
+    "\n",
+    "from pydantic import ValidationError, parse_obj_as\n",
+    "\n",
+    "jinja_pattern = r\"(?<=\\?)(?:\\w+)=\\w+&?\"\n",
```

Comment:
are you 100% sure that those are the only token names (I see `api_key` and `token` hard coded in the new version)? looks like we have others now, are they all new since September?
![image](https://user-images.githubusercontent.com/55149902/207376340-b36a6d4f-19c7-41b9-8526-3a524b53b0b4.png)

Reply:
Ah two of those are RT-only; if you look at just schedule downloads it's `token` and `api_key` which is also reflected in `status.csv` files.

![image](https://user-images.githubusercontent.com/4305366/207380739-4d9b9361-e90b-4c55-a7d2-94dd696b545e.png)